ui/static/js/bootstrap.js

```diff
@@ -28,6 +28,7 @@ document.addEventListener("DOMContentLoaded", function() {
     keyboardHandler.on("d", () => navHandler.fetchOriginalContent());
     keyboardHandler.on("f", () => navHandler.toggleBookmark());
     keyboardHandler.on("?", () => navHandler.showKeyboardShortcuts());
+    keyboardHandler.on("z c", () => navHandler.unsubscribeFromFeed());
```

Comment:
I don't really have any thoughts on the keyboard shortcuts, but I noticed that Gmail and Google Photos are using `#` to remove stuff. Perhaps, we could use the same. I guess they have pretty good UX folks.

Reply:
Good idea! Not many users would type `#` on accident, I'd bet.